README.md

```diff
@@ -6,10 +6,15 @@ A React component which makes it easy to create a directed graph editor without
 
 ## Installation
 
-```
+```bash
 npm install --save react-digraph
 ```
 
+If you don't have the following peerDependenies, make sure to install them:
+```bash
+npm install --save d3 radium react react-dom react-icons
```

Comment:
We've made a change to master that now includes d3, radium, and react-icons as core dependencies for react-digraph. Can you remove d3, radium, and react-icons from the list?